scripts/exp/ts-transform-import-path.js

```diff
@@ -46,7 +46,7 @@ function visitNodeAndChildren(node, context) {
 
 function visitNode(node) {
   let importPath;
-  if (ts.isImportDeclaration(node)) {
+  if (ts.isImportDeclaration(node) || ts.isExportDeclaration(node)) {
     const importPathWithQuotes = node.moduleSpecifier.getText();
```

Comment:
Seems to be causing a build error, need to do a null check on `moduleSpecifier`?

Reply:
yup, just fixed.